packages/ketcher-core/src/application/render/renderStruct.ts

```diff
@@ -24,6 +25,25 @@ export class RenderStruct {
     return struct;
   }
 
+  static removeSmallAttachemntPointLabelsInModal(render: Render, options) {
+    if (!options.labelInMonomerConnectionsModal) {
+      return;
+    }
+
+    const attachmentPointRegExp = /^R[1-8]$/;
```

Comment:
I wonder if it makes sense to store this as a member of the class itself to make it reusable for other methods... Just thinking. That may be redundant.